src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUploadOrchestrator.cs

```diff
@@ -318,5 +321,13 @@ private IEnumerable<UploadError> GetOverlappingErrors(OverlappingApprenticeship
             }
             return Enumerable.Empty<UploadError>();
         }
+
+        private void AssertCohortNotPaidForByTransfer(CommitmentView commitment)
```

Comment:
I personally don't like the name of this method, especially as 'Assert' has a specific meaning in C#. If the name was more descriptive, you could know what it did without looking at the code, e.g. ThrowIfCohortPaidForByTransfer.